exercicios/projeto/Account/Account.js

```diff
@@ -32,38 +43,61 @@ class Account {
     } else throw new Error("Não é possível depositar valores não numéricos.");
   }
 
-  saque(value) {
+  withdrawal(value) {
     if (typeof value === "number") {
       if (this.#saldo > value) {
```

Comment:
Sugestão: this.#saldo >= value, pois o saque pode ser realizado se o valor do saque for menor ou igual ao valor em conta.